windows.versions

```diff
@@ -1,5 +1,17 @@
 {
   "windows": [
+    {
+      "version": "2.31.8",
+      "build": "1",
+      "fluent-bit": "1.9.10",
+      "kinesis-plugin": "v1.10.1",
+      "firehose-plugin": "v1.7.1",
+      "cloudwatch-plugin": "v1.9.2",
+      "openssl": "3.0.7",
+      "flexBison": "2.5.22",
+      "latest": true,
```

Comment:
please ensure latest is set to false in 2.31.7.

Reply:
!!!!!